### DISCUSSION TITLE
can't display metrics in official ASP.NET OTEL (OpenTelemetry) Metrics grafana dashboard

### DISCUSSION THREAD

**Discussion by ravabl** [open-telemetry/opentelemetry-dotnet#4590] [Q&A] [answered]
Hi all!
Preconditions:
- OpenTelemetry .NET installed on all .net apps I need.
- metrics are visible on the ports I need
- targets in prometheus are configured and visible.
- already tried to load a dashboard with different versions of prometheus - both with the 1st, as indicated in the dashboard documentation, and in the latest one.

I am facing the following problem:

after importing grafana dashboard
ASP.NET OTEL (OpenTelemetry)
https://grafana.com/grafana/dashboards/17706-asp-net-hotel-metrics/
Metrics are not reflected in the dashboard.
The panels have been created, but there are no metrics in them.
The dashboard also has very "strange" variables.
![image](https://github.com/open-telemetry/opentelemetry-dotnet/assets/53080974/11fe80ec-1089-4297-bcdf-4eb35b31210b)
After their adjustment, jobs and hosts became available,
![image](https://github.com/open-telemetry/opentelemetry-dotnet/assets/53080974/7dbd003b-8feb-4855-8b98-92e03ba5f52a)

![image](https://github.com/open-telemetry/opentelemetry-dotnet/assets/53080974/ac73ff4f-93ca-4eb1-a1bf-19d47f293475)

but the metrics are still not available.
What could be the problem?
I think that the dashboard is not up-to-date and it hasn't been updated for the existing version of the exporter.
How can I solve this problem?
Graphs in prometheus are inconvenient for analysis.
Maybe one of the dashboard maintainers has a prerelease version of the dashboard?

**Comment by cijothomas** [comment, marked as answer]
This repo does not ship any Grafana dashboards, so we won't be able to help much. If the metrics are available in prometheus, then rest is beyond the scope of this repo.
